WMF Framework/Significant Events Endpoint/ArticleAsLivingDocViewModels.swift

```diff
@@ -187,6 +188,94 @@ public struct ArticleAsLivingDocViewModel {
         self.newChangesTimestamp = newChangesTimestamp
         self.lastUpdatedTimestamp = lastUpdatedTimestamp
     }
+
+    /// Collapses sequential sections that contain only small events into one section, including a date range that represents the collected events
+    static func collapseSmallEvents(from sections: [SectionHeader]) -> [SectionHeader] {
+        guard let dayMonthNumberYearDateFormatter = DateFormatter.wmf_monthNameDayOfMonthNumberYear(), let isoDateFormatter = DateFormatter.wmf_iso8601() else {
```

Comment:
If it's cool with you, I'd prefer to address this if necessary in the timestamp PR rather than muddying this PR - was already messing around with it and `Section` hashability again there.